static/files/sitemap.xml

```diff
@@ -839,12 +839,6 @@
         <changefreq>hourly</changefreq>
         <priority>0.5</priority>
       </url>
-      <url>
```

Comment:
Huh?

Reply:
@anthonydillon I removed that entry from the sitemap, as it's returning a 404. Should I have approached this problem differently?

Reply:
No just curious why